main.cpp

```diff
@@ -205,29 +260,45 @@ struct PersonTwo
     bool run(int speed);
     int jump(int crouch, int energy);
     bool playPiano(bool memoryRecal, int nerves);
+    void face();
 };
 
-bool PersonTwo::run(int speed)
+PersonTwo::PersonTwo()
+{
+    std::cout << "PersonTwo being constructed!" << std::endl; 
+}
+
+PersonTwo::Eye::Eye()
+{
+    std::cout << "The eye is being constructed!" << std::endl;
+}
+
+bool PersonTwo::run(int speed = 7)
 {
     return (speed > 5) ? true : false;
 }
 
-int PersonTwo::jump(int crouch, int energy)
+int PersonTwo::jump(int crouch = 9, int energy = 4)
 {
     return crouch * energy;
 }
 
-bool PersonTwo::playPiano(bool memoryRecal, int nerves)
+bool PersonTwo::playPiano(bool memoryRecal = true, int nerves = 11)
```

Comment:
Default arguments should be defined where the function is declared, not where the function is defined. 
- [ ] revise